polyphemus/Makefile

```diff
@@ -1,5 +1,15 @@
 include ../make-base/stubs.mk
 
+.projects-mark:
+	git clone git@github.com:mountetna/redcap-projects.git lib/etls/redcap/projects || CLONE_FAILED=1
+	if [ -z "$(CLONE_FAILED)" ]; then \
+		echo "Could not clone REDCap projects -- do you have the right permissions?"; \
+	fi
+	@ touch .projects-mark
```

Comment:
Do we want to touch the mark on failure?  We should maybe put this into an else clause.